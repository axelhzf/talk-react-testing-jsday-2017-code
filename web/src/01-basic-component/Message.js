export function Message() {
  return 'Hello world';
}
