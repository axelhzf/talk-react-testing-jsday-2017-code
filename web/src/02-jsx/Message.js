import * as React from 'react';

export function Message() {
  return (
    <div className="container">
      <h1>Hello</h1>
      <h2>World</h2>
    </div>
  );
}
