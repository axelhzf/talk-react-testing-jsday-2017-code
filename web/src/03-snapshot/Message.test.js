import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import { Message } from './Message';

describe('02-jsx', () => {
  it('should render a message', () => {
    const renderer = ReactTestRenderer.create(<Message />);
    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
