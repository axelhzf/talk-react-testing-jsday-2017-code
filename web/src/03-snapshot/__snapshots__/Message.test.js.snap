// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`02-jsx should render a message 1`] = `
<div
  className="container"
>
  <h1>
    Hello
  </h1>
  <h2>
    World
  </h2>
</div>
`;
